napari_allencell_annotator/view/images_view.py

```diff
@@ -224,12 +224,18 @@ def _add_selected_dir_to_ui(self, dir_path: Path) -> None:
         dir_list : List[Path]
             The input list with dir[0] holding directory name.
         """
-        all_files_in_dir: list[Path] = FileUtils.get_sorted_files_in_dir(dir_path)
+        # ome.zarr folder
+        if dir_path.suffix == ".zarr":
+            self._add_selected_files([dir_path])
 
-        if len(all_files_in_dir) < 1:
-            self.viewer.alert("Folder is empty")
+        # other
         else:
-            self._add_selected_files(all_files_in_dir)
+            all_files_in_dir: list[Path] = FileUtils.get_sorted_dirs_and_files_in_dir(dir_path)
+
+            if len(all_files_in_dir) < 1:
+                self.viewer.alert("Folder is empty")
+            else:
+                self._add_selected_files(all_files_in_dir)
```

Comment:
why's this handled here instead of where you select a single image?

Reply:
fixed in https://github.com/aics-int/napari-allencell-annotator/pull/79/commits/46825f8911afea51f24c4ed486d404980a5849b9